charts/cluster-api-runtime-extensions-nutanix/templates/csi/nutanix/manifests/helm-addon-installation.yaml

```diff
@@ -10,6 +10,7 @@ data:
   values.yaml: |-
     # The Secret containing the credentials will be created by the handler.
     createPrismCentralSecret: false
+    createSecret: false
```

Comment:
why this change got introduced?

Reply:
Let me leave a comment.

Reply:
Added a comment in https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/pull/814/commits/6140ddafc038ca024c662629c98b75881fcaca69.

It was failing with
```
(PE credentials) are required, if createSecret is set to true
```